tests/phpunit/includes/testcase-emails.php

```diff
@@ -14,22 +14,4 @@ public static function set_up_before_class() {
 		 */
 		bp_core_install_emails();
 	}
-
-	public static function tear_down_after_class() {
-		$emails = get_posts( array(
-			'fields'           => 'ids',
-			'post_status'      => 'any',
-			'post_type'        => bp_get_email_post_type(),
-			'posts_per_page'   => 200,
-			'suppress_filters' => false,
-		) );
-
-		if ( $emails ) {
-			foreach ( $emails as $email_id ) {
-				wp_delete_post( $email_id, true );
-			}
-		}
-
-		parent::tear_down_after_class();
-	}
```

Comment:
I just need a confirmation: these emails are automatically removed at the next time we launch the tests, that's why you think it's not needed to removed them here?

Reply:
@imath That's correct. All posts are removed at tear_down_after_class. See https://github.com/WordPress/wordpress-develop/blob/859263db0657a7f536fdeefe60a1225ceb6fd4f7/tests/phpunit/includes/functions.php#L110